default yarn install fails

## Bug Report

### Steps to Reproduce:
run 
```bash
yarn
```

Postinstall task of `sharp` (dependency of `literally-cli` tries to run native build of vips, fails for me(macos)

../src/common.cc:23:10: fatal error: 'vips/vips8' file not found

I can install with `--ignore-scripts` and go on to `yarn build` and all seems ok

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -15,7 +15,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [12.x]
+        node-version: [16.x]
 
     steps:
     - uses: actions/checkout@v2

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "url": "https://github.com/finos/regular-table"
     },
     "engines": {
-        "node": ">=10.18.1"
+        "node": ">=16"
     },
     "license": "Apache-2.0",
     "main": "dist/umd/regular-table.js",
@@ -70,7 +70,7 @@
         "jest-puppeteer-docker": "^1.4.2",
         "jsdoc-to-markdown": "^6.0.1",
         "less": "^3.9.0",
-        "literally-cli": "^0.0.9",
+        "literally-cli": "^0.0.10",
         "marked-ast-markdown": "^2.1.0",
         "npm-run-all": "^4.1.3",
         "prettier": "^2.0.5",

